www/grpc/blockchain_test.go

```diff
@@ -179,6 +180,40 @@ func TestGetAccount(t *testing.T) {
 	})
 	assert.Nil(t, conn.Close(), "Error closing connection")
 }
+
+func TestGetAccountByNumber(t *testing.T) {
+	conn, client := testBlockchainClient(t)
+	acc, _ := tMockState.TestStore.AddTestAccount()
+
+	t.Run("Should return error for non-parsable address ", func(t *testing.T) {
+		res, err := client.GetAccountByNumber(tCtx,
+			&pactus.GetAccountByNumberRequest{Number: 0})
+
+		assert.Error(t, err)
+		assert.Nil(t, res)
+	})
+
+	t.Run("Should return nil for non existing account ", func(t *testing.T) {
+		res, err := client.GetAccountByNumber(tCtx,
+			&pactus.GetAccountByNumberRequest{Number: rand.Int31()})
```

Comment:
```suggestion
			&pactus.GetAccountByNumberRequest{Number: rand.Int32()})
```

Reply:
I like the idea of random number here.
 could you please update the `TestGetValidatorByNumber` to use random number instead of  `val1.Number() + 1`.

Reply:
@b00f practically Int31 returns a non-negative pseudo-random 31-bit integer as an int32 from the default Source.

Reply:
use our `util` package: `util.RandInt32`